src/main/java/seedu/address/model/pet/Species.java

```diff
@@ -4,39 +4,39 @@
  * Represents the species (kind) of a pet.
  */
 public class Species {
-    private final String species;
+    private final String value;
 
     /**
      * Constructs the Species object.
      *
-     * @param species The string representation of the species.
+     * @param value The string representation of the species.
      */
-    public Species(String species) {
-        if (species == null) {
-            this.species = "";
+    public Species(String value) {
```

Comment:
Can you update the method name in the Species test so that the test case will pass and CI will pass? thanks!